src/ResourceManager/DataLakeStore/Commands.DataLakeStore/Commands/SetAzureRmDataLakeStoreAccount.cs

```diff
@@ -44,24 +44,14 @@ public class SetAzureDataLakeStoreAccount : DataLakeStoreCmdletBase
         public Hashtable Tags { get; set; }
 
         [Parameter(ValueFromPipelineByPropertyName = true, Position = 3, Mandatory = false,
-            HelpMessage = "Indicates what type of encryption to provision the account with, if any.")]
+            HelpMessage = "Optionally enable/disable the existing trusted ID providers.")]
         [ValidateNotNull]
-        public EncryptionConfigType? Encryption { get; set; }
+        public TrustedIdProviderState? TrustedIdProviderState { get; set; }
 
         [Parameter(ValueFromPipelineByPropertyName = true, Position = 4, Mandatory = false,
-            HelpMessage = "If the encryption type is User assigned, this is the key vault the user wishes to use")]
+            HelpMessage = "Optionally enable/disable existing firewall rules.")]
         [ValidateNotNull]
-        public string KeyVaultId { get; set; }
-
-        [Parameter(ValueFromPipelineByPropertyName = true, Position = 5, Mandatory = false,
-            HelpMessage = "If the encryption type is User assigned, this is the key name in the key vault the user wishes to use")]
-        [ValidateNotNull]
-        public string KeyName { get; set; }
-
-        [Parameter(ValueFromPipelineByPropertyName = true, Position = 6, Mandatory = false,
-            HelpMessage = "If the encryption type is User assigned, this is the key version of the key the user wishes to use")]
-        [ValidateNotNull]
-        public string KeyVersion { get; set; }
```

Comment:
Same issue as the others. When adding tests, I discovered that this wasn't actually supported and will always fail. Removing it protects both us and our customers from assuming they can do things that are not supported. Unfortunately we should have caught this previously, but we did not have the bandwidth (or a good way) to test it until this cycle, where I have found the bugs before customers and am proactively fixing them.